pkg/scheduler/internal/cache/node_tree.go

```diff
@@ -111,7 +111,7 @@ func (nt *NodeTree) removeNode(n *v1.Node) error {
 					nt.removeZone(zone)
 				}
 				klog.V(5).Infof("Removed node %v in group %v from NodeTree", n.Name, zone)
-				nt.NumNodes--
+				nt.numNodes--
```

Comment:
noit: Shouldn't log statement be after `numNodes--`

Reply:
They are always being done before the changes, I did not modify that